lib/offline/background.ts

```diff
@@ -0,0 +1,21 @@
+import { jsonApiPost } from "../axios";
+import { processJournal, processTechnical } from "../process";
+import { offlineDB } from "./db";
+
+export async function submitOfflineJournals() {
+  const data = await offlineDB.farmerJournal.toArray();
+  for (const entry of data) {
+    console.log("post journal", entry.id);
```

Comment:
Do we want to eventually remove `console.log` statements and use a separate logger?

Reply:
removed all `console.log`